scheds/rust/scx_p2dq/src/bpf/main.bpf.c

```diff
@@ -330,16 +332,15 @@ static struct llc_ctx *pick_two_llc_ctx(struct llc_ctx *cur_llcx, struct llc_ctx
 		cur_queued += scx_bpf_dsq_nr_queued(cur_dsq_id);
 	}
 
-	if (min_nr_queued_pick2 > 0 &&
-	    cur_queued < min_nr_queued_pick2)
+	if (min_nr_queued_pick2 > 0 && cur_queued < min_nr_queued_pick2)
 		return NULL;
 
 	left_load = left->load;
 	right_load = right->load;
 
 	// If the current LLCs has more load don't try to pick2.
 	cur_load += (lb_slack_factor * cur_load) / 100;
-	if ((nr_llcs > 2 && cur_load > left_load && cur_load > right_load))
+	if ((nr_llcs > 2 && (cur_load > left_load || cur_load > right_load)))
```

Comment:
This is an interesting change. Previously we'd have picked from the more heavily loaded option of the two, now we pick from neither if we're in the middle. Seems to make sense and reduce migrations.